packages/manager/modules/iam/src/routes/policyAdd.route.js

```diff
@@ -0,0 +1,24 @@
+import { policyAdd } from '@iam/components';
+import { asParams, asResolve } from '@iam/resolves';
+
+export const name = 'policyAdd';
+export const params = [];
+
+export const state = () => ({
+  url: '/policy/add',
+  component: policyAdd.name,
+  params: asParams(params),
+  resolve: {
+    ...asResolve(policyAdd.resolves),
+    breadcrumb: /* @ngInject */ ($translate) =>
+      $translate.instant('iam_policy_add_title'),
+  },
+});
+
+export const children = [];
+
+export default {
+  name,
+  state,
+  children,
```

Comment:
This can be removed safely if there are no children routes

Reply:
It's fixed.